helm/cert-exporter/templates/vpa-deployment.yaml

```
@@ -0,0 +1,23 @@
+{{- if .Values.deployment.autoscaling.enabled }}
```

```suggestion
{{- if .Values.deployment.autoscaling.enabled -}}
```

Not 100% sure about that, maybe you want to `helm template` to check, but without this you would get an empty line.

---

Ah, I think this only makes a difference if you have two conditions in the beginning. This is why I told "not sure". I saw Helm rendering an empty line once due to ✨ _circumstances_ ✨.